multi-node/vagrant/Vagrantfile

```diff
@@ -8,7 +8,7 @@ require 'yaml'
 
 Vagrant.require_version ">= 1.6.0"
 
-$update_channel = "stable"
+$update_channel = "alpha"
```

Comment:
Yeah, part of the motivation is that alpha makes sense for Vagrant clusters anyway and it avoids having to write silly version logic for people using rkt.